<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CreateRect">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>550</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Rectangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>550</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Rectangle</string>
    </property>
   </widget>
   <widget class="PygletWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1441</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>995</width>
      <height>232</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Train List</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string notr="true">Headcode </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">For</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">To</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string notr="true">Next</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Center</string>
           </property>
          </column>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget_2">
          <row>
           <property name="text">
            <string/>
           </property>
          </row>
          <column>
           <property name="text">
            <string notr="true">Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Location</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Center</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Timing</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Simulation Time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdNumber"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Pause</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>1x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>2x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>5x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>10x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>20x</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>openGLWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>label_2</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>CreateRect</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Simulation"/>
    <addaction name="actionOpen_Map"/>
    <addaction name="actionOpen_Timetable"/>
    <addaction name="actionSave_Scenario"/>
    <addaction name="actionOpen_Scenario"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionActual_Size"/>
    <addaction name="actionToggle_Grid"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionErrors"/>
    <addaction name="actionWarnings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSimulation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionRed"/>
   <addaction name="actionyellow"/>
   <addaction name="actiondYellow"/>
   <addaction name="actionGreen"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_Track"/>
   <addaction name="separator"/>
   <addaction name="actionAuto_Track"/>
   <addaction name="actionRoute_Train"/>
  </widget>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionyellow">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actiondYellow">
   <property name="text">
    <string>Double Yellow</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionToggle_Track">
   <property name="text">
    <string>Toggle Point</string>
   </property>
  </action>
  <action name="actionAuto_Track">
   <property name="text">
    <string>Auto Track</string>
   </property>
  </action>
  <action name="actionRoute_Train">
   <property name="text">
    <string>Route Train</string>
   </property>
  </action>
  <action name="actionNew_Simulation">
   <property name="text">
    <string>New Simulation</string>
   </property>
  </action>
  <action name="actionOpen_Map">
   <property name="text">
    <string>Open Map</string>
   </property>
  </action>
  <action name="actionOpen_Timetable">
   <property name="text">
    <string>Open Timetable</string>
   </property>
  </action>
  <action name="actionSave_Scenario">
   <property name="text">
    <string>Save Scenario</string>
   </property>
  </action>
  <action name="actionOpen_Scenario">
   <property name="text">
    <string>Open Scenario</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionActual_Size">
   <property name="text">
    <string>Actual Size</string>
   </property>
  </action>
  <action name="actionToggle_Grid">
   <property name="text">
    <string>Toggle Grid</string>
   </property>
  </action>
  <action name="actionErrors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Errors</string>
   </property>
  </action>
  <action name="actionWarnings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warnings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PygletWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>pygletWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
